matsuoharuki/pbl2_group2
# Front routing

In this pull request:
 
- Merged backend and frontend code
- Created frontend routing to send a request to the backend to register a user
- Removed single client connection and added connection pool to allow multiple transactions
- Fixed issue where the exception message is empty when a query failed
- Changed backend running port to 5000 instead of 3000 because there were some port already used issues

To test:
- Start front docker and back docker (back docker now uses port 5000 so use: docker run -it -p 5000:5000 pbl2-back)
- go to localhost:8080/signup in browser
- Register a new user

Test success:
- A new user is created in the database
Test failure:
- No user is created and the exception message is shown in the console
